### PR TITLE
tstrait-news

### PR DESCRIPTION
**Pull request by daikitag**
@hyanwong Does this look good? Thank you very much for writing the news article, and I'm very excited about it!

### CODE REVIEW

**Comment by daikitag** [comment]
@hyanwong Does this look good?

**Comment by hyanwong** [comment]
Seems GTM. Lets see what @benjeffery thinks.

File: _news/20230920-tstrait-0.0.1.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  simulator leverages the succinct tree sequence format to do so
+  efficiently for large numbers of genomes.
+  
+  The new software, tstrait, takes an existing tree sequence containing
+  mutations, and simulates a quantitative phenotypic trait whose value
+  is partially determined by a selected set of causal mutations in the
+  tree sequence.
+
+  This is an initial 0.0.1 release.
+image: ../assets/images/news/20230905-tstrait/daiki.jpg
```
Comment (hyanwong) [inline _news/20230920-tstrait-0.0.1.md:15]: Should be `image: ../assets/images/news/20230920-tstrait-0.0.1/daiki.jpg`

File: _news/20230920-tstrait-0.0.1.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+timestamp: 2023-09-20
+author: Yan Wong
+---
+
+# Ts-trait: new software for simulating quantitative traits (phenotypes)
+
+![Daiki Tagami]({{ '/assets/images/news/20230920-tstrait-0.0.1/daiki.jpg' |  relative_url}}){:class="news-float-left"}
```
Comment (hyanwong) [inline _news/20230920-tstrait-0.0.1.md:22]: Makes the picture too big. Try using e.g. `{:class="news-float-left" style="width: 8em"}`

File: _news/20230920-tstrait-0.0.1.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+Daiki Tagami, a student in the Kelleher group, has built [tstrait](https://tskit.dev/software/tstrait.html),
+a software package to layer simulated quantitative phenotype traits on top of an existing tree sequence. This
+should be useful, for example, to create large simulated datasets for testing genome-wide association (GWAS)
+methods.
+
+## Example code
+
```
Comment (hyanwong) [inline _news/20230920-tstrait-0.0.1.md:30]: Add `{:style="break: both"}` as the line immediately after the `## Example code` line.

**Review by hyanwong — APPROVED** [review]
Just a few minor things.

**Comment by hyanwong** [comment]
Oh, and (sorry!) a last thing: do you want to add the plot itself to the page. See e.g. https://tskit.dev/news/20221025-tsinfer-0.3.0.html which has a few embedded plot outputs.

**Comment by daikitag** [comment]
I think it would be great to add a plot to the news page. Do you think it is possible to add the histogram that is produced in the example code to the website?

**Comment by daikitag** [comment]
@hyanwong I think there is a warning message showing up in the html theme of `conf.py` file. Would it be possible for you to check it, as it is unrelated to the tstrait news update?

**Comment by hyanwong** [comment]
> I think it would be great to add a plot to the news page. Do you think it is possible to add the histogram that is produced in the example code to the website?

I think you can just save it to PNG and put it in the same folder as your photo?

**Comment by benjeffery** [comment]
Let me know when you're done here @daikitag - there are some issues with doc builds that I'm fixing that need to go in.

**Comment by daikitag** [comment]
@benjeffery I think it is done now. Would it be possible for you to update the changes? Thank you for your help.

**Comment by benjeffery** [comment]
Rebased

**Comment by hyanwong** [comment]
I'm not sure the link in the `description: |` field plays nicely: but otherwise LGTM. @benjeffery - do you want to remove the link (`[tstrait](https://tskit.dev/software/tstrait.html)` -> `tstrait`) and merge?

**Comment by benjeffery** [comment]
Removed from description and moved to text. Documentation link fixed.

**Pull request merged**